docs/pages/blog/2020-developer-survey-results.md

```diff
@@ -179,10 +179,38 @@ As the answers to this questions were pretty different, we grouped them into dif
   <tr><th>3</th><th> grid - improve</th><tr>
 </table>
 
+### Comparison with last year
+
+There are a couple of noticeable differences compared to last year.
+Some can be explained by our work, others by an evolution of the ecosystem.
+Each item is prefixed by the multiplicator factor of pain point between 2019 and 2020.
+
+Decreasing pain:
+- x0: Slider. No requests, the problem is almost solved since the introduction of the range feature.
+- x0.1: Strict mode. We did a lot for it this year. However, since Create React App has made this mode a default, we have seen a lot more requests for it.
+- x0.1: Autocomplete. We did a lot for it this year. The component will be moved to the core in v5.
+- x0.2: Fewer breaking changes. Only releasing minor versions under v4 for over a year helps a lot. However, we still need to be careful with CSS changes.
+- x0.3: Accessibility. We did a lot for it this year. One of our best leverage has been GitHub's issue opened by a11y expert, often coming from large companies using Material-UI at scale.
+- x0.4: Material design. We didn't do much for it this year, at least not as much as we could have. Maybe fewer people care? It seems that we resonate better with developers going for custom design systems.
```

Comment:
```suggestion
- x0.4: Material design. We didn't do much for it this year, at least not as much as we could have. Maybe the reduction is because fewer people care? It seems that we start to resonate more with developers building custom design systems.
```